common/api-review/firestore-lite.api.md

```diff
@@ -19,11 +19,15 @@ export type AddPrefixToKeys<Prefix extends string, T extends Record<string, unkn
 
 // @public
 export class AggregateField<T> {
+    readonly aggregateType: AggregateType;
```

Comment:
the `field` public member is missing

Reply:
Resolved in out of band conversation. Not including the `field` public memeber.